main.go

```diff
@@ -66,6 +67,11 @@ func main() {
 							Usage: "Optional logfile to send runtime logs to",
 						},
 						cli.StringFlag{
+							Name:  "static-dir, s",
+							Usage: "Any static assets (e.g. CSS/Javascript/HTML) files located in this directory will be presented at /",
```

Comment:
Can you eloborate what `/` refers to? Something like "mounted at `/` of API gateway URL"